net/ddns-scripts/Makefile

```diff
@@ -362,6 +372,15 @@ define Package/ddns-scripts-services/install
 endef
 
 
+define Package/ddns-scripts-utils/install
+	$(INSTALL_DIR) $(1)/usr/lib/ddns
+	$(INSTALL_BIN) ./samples/slaac_sample.sh \
+		$(1)/usr/lib/ddns/ddns-slaac.sh
+	$(INSTALL_BIN) ./samples/getlocalip_sample.sh \
+		$(1)/usr/lib/ddns/ddns-getlocalip.sh
```

Comment:
Now I haven't read properly where it is installed. I thought it was installed under `/bin` or `/usr/bin`. If it is installed under `/usr/lib/ddns`, we don't necessarily have to change the name.

The question now is whether it should be installed under `/bin` or `/usr/bin`, or whether we should put it under `/usr/lib/ddns` as you already have done.

For which use case are the scripts useful?

Reply:
So, I have a raspberry pi connected to my openwrt router, and it gets its ipv6 address via slaac.
In order to have the ddns record point to my raspberry pi I have:

```
config service 'myddns_ipv6'
        option use_ipv6 '1'
        option enabled '1'
        option lookup_host '...'
        option service_name 'no-ip.com'
        option domain '...'
        option username '...'
        option password '...'
        option use_syslog '2'
        option check_unit 'minutes'
        option force_unit 'minutes'
        option retry_unit 'seconds'
        # this points to openwrt's public ip address
        # option ip_source 'network'
        # option ip_network 'wan6'
        option ip_source 'script'
        option ip_script '/usr/lib/ddns/slaac_sample.sh wan b8:27:eb:13:9a:2b' // <-- this
        option interface 'wan6'
```

I think `/usr/lib` makes more sense, they are not meant to be used outside the context of being an argument to the `ip_script` option. The [comment](https://github.com/openwrt/packages/blob/f3a6491674ab451535e392b63e94a545abf3f292/net/ddns-scripts/samples/slaac_sample.sh#L9) on the files also makes this suggestion.
I'm also thinking what's the best naming here... maybe dropping the `ddns-` prefix will make most sense?

Reply:
Make it so

Reply:
done!